packages/cli/src/commands/init/__tests__/editTemplate.test.ts

```diff
@@ -88,10 +98,18 @@ skipIfNode20('should edit template', async () => {
     'utf8',
   );
 
-  expect(snapshotDiff(oldCFile, newCFile, {contextLines: 1})).toMatchSnapshot();
   expect(
     snapshotDiff(oldJavaFile, newJavaFile, {contextLines: 1}),
   ).toMatchSnapshot();
+
+  expect(snapshotDiff(oldCFile, newCFile, {contextLines: 1})).toMatchSnapshot();
+  expect(
+    snapshotDiff(oldXmlFile, newXmlFile, {contextLines: 1}),
+  ).toMatchSnapshot();
+  console.log({
+    fixtureTree,
+    transformedTree,
+  });
   expect(
     snapshotDiff(fixtureTree.map(slash), transformedTree.map(slash), {
```

Comment:
fixtureTree likely needs to be sorted alphabetically, I suppose this is why snapshot differs between mac and linux